Update list_alerts.py (#3)
SEKOIA-IO/connector-sekoia-io-xdr: README.md

# connector-sekoia-io-xdr

> In order to enable our client SNS security to make full use of our XDR platform, we were asked to create this SEKOIA.IO python connector in FortiSOAR.

Reviewer: ```suggestion
This connector enable you to make full use of the SEKOIA.IO XDR platform.
```

> #### It includes ths following actions:

Reviewer: ```suggestion
It includes the following actions:

```
